### PR TITLE
Project Readme

### PR DESCRIPTION
**Pull request by dometto**
How does this look as a structure (and start) of the Readme?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,149 @@
+# CuttingEdge -- Simple, self-hosted dependency monitoring
```
Comment (bartkamphorst) [inline README.md:1]: Nice slogan.

File: README.md [hunk 2]
```diff
@@ -1,95 +1,149 @@
...
+
 [![Build Status](https://travis-ci.org/repotag/cutting_edge.svg?branch=master)](https://travis-ci.org/repotag/cutting_edge)
 [![Coverage Status](https://coveralls.io/repos/github/repotag/cutting_edge/badge.svg?branch=master)](https://coveralls.io/github/repotag/cutting_edge?branch=master)
 [![Cutting Edge Dependency Status](https://dometto-cuttingedge.herokuapp.com/github/repotag/cutting_edge/svg 'Cutting Edge Dependency Status')](https://dometto-cuttingedge.herokuapp.com/github/repotag/cutting_edge/info)
 
-# CuttingEdge
+CuttingEdge monitors the status of the dependencies of your projects and lets you know when any of them go out of date. It:
+
+* Generates badge images that you can include in your projects' Readme, like the one above!
+* Can send you e-mail when the status of a project's dependencies changes
```
Comment (bartkamphorst) [inline README.md]: `email`.

File: README.md [hunk 3]
```diff
@@ -1,95 +1,149 @@
...
+* Serves a simple [info page](https://dometto-cuttingedge.herokuapp.com/github/repotag/cutting_edge/info) detailing the status for each project
```
Comment (bartkamphorst) [inline README.md]: the status `of` each project.

File: README.md [hunk 4]
```diff
@@ -1,95 +1,149 @@
...
+If you want to monitor dependencies in a private (e.g. GitHub or Gitlab) project, you can instruct CuttingEdge to use an API token for accessing the dependency files. In `projects.yml`:
+
+```yaml
+github:
+   secret-org:
+     secret-repo:
+       auth_token: 'mysecrettoken'
+```
+
+For info on geneating API tokens, see:
```
Comment (bartkamphorst) [inline README.md]: `generating` (typo).

File: README.md [hunk 5]
```diff
@@ -1,95 +1,149 @@
...
 
-Redis::Objects.redis = Redis.new
-gem_dependencies = Redis::Value.new(gem.identifier)
+This approach is especially useful on Heroku, where you can use [Heroku config variables](https://devcenter.heroku.com/articles/config-vars).
 
-puts gem_dependencies.value # Currently nil
+# Refreshing dependency status through git hooks
 
-DependencyWorker.perform_async(gem.identifier, gem.gemspec_location, gem.gemfile_location) # Fire up a Sidekiq job
+CuttingEdge by default checks whether the status of your dependencies has changed once every hour. However, if you wish, you can also setup hooks so that dependency status is checked (for instance) whenever you make a commit to your project.
```
Comment (bartkamphorst) [inline README.md:145]: This is cool. Have we tested this yet?

**Review by bartkamphorst — CHANGES_REQUESTED** [review]
I like the structure!

File: README.md (hunk 5 above)
Comment (dometto) [reply README.md:145]: No, although the`refresh` route is tested. Once we have an up to date server running on Heroku, I'll test it with gollum!

**Comment by dometto** [comment]
Thanks for the pointers! Added some more documentation. If you have a look at the new section, I think I'll merge this and then we can add/change sections more iteratively.

File: README.md [hunk 6]
```diff
@@ -1,95 +1,172 @@
...
+  * GitHub
+  * Gitlab (both gitlab.com and self-hosted instances)
+  * Gitea (self-hosted)
+  * Both public and [private repositories](#Authorization-and-private-repositories)
+
+Moreover, CuttingEdge is lightweight and easy to deploy: 
+
+* No database required
+* Simple configuration through a `projects.yml` file
+* Requires relatively few resources (~120MB RAM), so..
```
Comment (bartkamphorst) [inline README.md]: The ellipsis is three dots 😉

**Review by bartkamphorst — APPROVED** [review]
Feel free to merge. 👍

File: README.md (hunk 6 above)
Comment (dometto) [reply README.md]: I stand corrected!

**Pull request merged**